enso/config.py

```diff
@@ -83,27 +89,35 @@
 EXPERIMENTS = {
     # "FinetuneSequenceLabel",
     # "IndicoSequenceLabel"
-    "LogisticRegressionCV",
-    "KNNCV",
-    "TfidfKNN",
-    "TfidfLogisticRegression",
-    "KCenters",
-    "TfidfKCenters"
+    "LogisticRegressionCVRationalized",
+    "FinetuneSeqBaselineRationalized",
+    "FinetuneClfBaselineRationalized",
+#    "LogisticRegressionCV",
+#    "KNNCV",
+#    "TfidfKNN",
+#    "TfidfLogisticRegression",
+#    "KCenters",
+#    "TfidfKCenters"
     # "SupportVectorMachineCV",
 }
 
 # Metrics to compute
-METRICS = {"Accuracy", "MacroRocAuc"}
+METRICS = {
+#    "Accuracy",
+    "AccuracyRationalized",
+    "MacroRocAucRationalized",
+#    "MacroRocAuc"
+}
 
 # Test setup metadata
 TEST_SETUP = {
-    "train_sizes": iter([50, 60, 75, 100, 125, 150, 175, 200]),
+    "train_sizes": [20, 40, 60, 80, 100, 150],
```

Comment:
Some of these dataset should have training sizes > 150 examples yea?  It's ok if some datasets don't run at the upper end of that.